src/test/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/VisitorInformationRepositoryTest.java

```diff
@@ -19,19 +19,19 @@
 @AutoConfigureTestDatabase(replace = NONE)
 @Import({AuthenticationFacade.class, AuditorAwareImpl.class})
 @WithMockUser
-public class    VisitorInformationRepositoryTest {
+public class VisitorInformationRepositoryTest {
 
     @Autowired
     private VisitorRepository repository;
 
     @Test
     public void findAllByVisitId() {
-        var visits = repository.findAllByVisitId(-15L);
+        final var visits = repository.findAllByVisitId(-15L);
 
-        assertThat(visits).hasSize(3);
```

Comment:
no point testing the size when we are testing all the values in the table anyway